frontend/public/components/_quota.scss

```diff
@@ -21,41 +18,16 @@
   padding-left: 34px;
 }
 
-.co-resource-quota-scope__description {
-  font-size: $font-size-small;
+.quota-dashboard-scopes {
```

Comment:
Add `co-` so the classes are consistent?

Reply:
fixed